sentry/src/main/java/io/sentry/SentryBaseEvent.java

```diff
@@ -185,8 +185,15 @@ public void setThrowable(final @Nullable Throwable throwable) {
     return tags;
   }
 
-  public void setTags(Map<String, String> tags) {
-    this.tags = tags;
+  public void addTags(Map<String, String> tags) {
```

Comment:
I believe its better to explicit here that we are not actually "setting" tags but adding tags from one collection to existing tags. 

This also means that this change is breaking on the compilation level. I am not sure if this is what we want to do or we want to keep it backward compatible @marandaneto @bruno-garcia

Reply:
that does against our unified API, I do agree its a convenience but the public API already exposes `setField(x, y)` and `setFields(fields)` so I'd stick with the unified API.
One of them only tests the value to a specific key and the other replace everything.

For Kotlin, when we work on the Kotlin extensions, we could have extension methods so the syntax would be Kotlin friendlier eg `keys[x] = y`

Reply:
indeed, I totally agree with the Java convention point of view, actually, I thought those methods were in the Scope but actually, they are in the SentryBaseEvent so these additional methods actually make sense to me, unless @bruno-garcia is against them.

Reply:
Decided to drop `addXXX` method and in `setFields(fields)` follow Java convention to replace the collection just that instead of assigning the value given by parameter, new mutable collection is created containing all elements from the parameter.

Reply:
nice, thanks